jobs/JGDAS_ATMOS_VERFRAD

```diff
@@ -57,31 +51,26 @@ export TANKverf_radM1=${TANKverf_radM1:-${COM_ATMOS_RADMON_PREV}}
 if [[ ! -d ${TANKverf_rad} ]]; then mkdir -p -m 775 ${TANKverf_rad} ; fi
 if [[ ! -d ${TANKverf_radM1} ]]; then mkdir -p -m 775 ${TANKverf_radM1} ; fi
 
-########################################
-# Set necessary environment variables
-########################################
-export RAD_AREA=${RAD_AREA:-glb}
-
+#############################################
+#  Filenames
 export biascr=${biascr:-${COM_ATMOS_ANALYSIS}/gdas.t${cyc}z.abias}
 export radstat=${radstat:-${COM_ATMOS_ANALYSIS}/gdas.t${cyc}z.radstat}
+export satype_file=${satype_file:-${FIXgdas}/gdas_radmon_satype.txt}
 
-echo " "
-echo "JOB HAS STARTED"
-echo " "
-
+#  Other variables
+export RAD_AREA=${RAD_AREA:-glb}
+export MAKE_CTL=${MAKE_CTL:-1}
+export MAKE_DATA=${MAKE_DATA:-1}
+export USE_ANL=${USE_ANL:-1}
+export PDATE=${PDY}${cyc}
+export DO_DIAG_RPT=${DO_DIAG_RPT:-1}
+export DO_DATA_RPT=${DO_DATA_RPT:-1}
```

Comment:
Do we really need pass-through capability here?
For e.g. `RAD_AREA="glb"` should be sufficient.  Some of these could be even elevated to the configs.
`PDATE` can be exported at line 36 replacing `pdate`.  Using 2 variables with the same name one uppercase and another lowercase should be unnecessary.

Reply:
Moving settings to configs. The uppercase `PDATE` is current cycle, while the lower case `pdate` was the prior cycle but is now being changed to `GDATE`.